pyproject.toml

```
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "gvm-libs"
+version = "21.10.0.dev1"
```

I guess this version is wrong.
```suggestion
version = "21.04.1.dev1"
```